cmd/kube-controller-manager/app/certificates.go

```diff
@@ -36,12 +36,12 @@ import (
 func startCSRSigningController(ctx ControllerContext) (http.Handler, bool, error) {
 	gvr := schema.GroupVersionResource{Group: "certificates.k8s.io", Version: "v1", Resource: "certificatesigningrequests"}
 	if !ctx.AvailableResources[gvr] {
-		klog.Warningf("Resource %s is not available now", gvr.String())
+		klog.InfoS("Resource is not available now", "resourceName", gvr.String())
```

Comment:
minor: Would you consider to use resource as the key since it not only contains resourceName?